docs/src/auth.md

```diff
@@ -36,6 +36,30 @@ await page.click('text=Submit');
 // Verify app is logged in
 ```
 
+```python-async
+page = await context.new_page()
```

Comment:
noticed some inconsistencies with the code from above of this block:

![image](https://user-images.githubusercontent.com/1477453/107073649-4a391900-67f0-11eb-9d23-65e514ec6fa0.png)

Reply:
`newPage`/ `new_page`